src/component/Footer.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import {makeStyles} from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Box from '@material-ui/core/Box';
+
+function Copyright() {
+    return (
+        <Typography variant="body2" color="textSecondary" align="center">
+            {'Copyright © '}
+            <Link color="inherit" href="https://material-ui.com/">
+                JobFinder FPT Team
+            </Link>{' '}
+            {new Date().getFullYear()}
+            {'.'}
+        </Typography>
+    );
+}
+
+const useStyles = makeStyles((theme) => ({
+    '@global': {
+        ul: {
+            margin: 0,
+            padding: 0,
+            listStyle: 'none',
+        },
+    },
+    footer: {
+        borderTop: `1px solid ${theme.palette.divider}`,
+        marginTop: theme.spacing(8),
+        paddingTop: theme.spacing(3),
+        paddingBottom: theme.spacing(3),
+        [theme.breakpoints.up('sm')]: {
+            paddingTop: theme.spacing(6),
+            paddingBottom: theme.spacing(6),
+        },
+    },
+}));
+
+
+const footers = [
+    {
+        title: 'Về chúng tôi',
+        description: ['Team', 'History', 'Contact us', 'Locations'],
+    },
+    {
+        title: 'Tính năng',
+        description: ['Cool stuff', 'Random feature', 'Team feature', 'Developer stuff', 'Another one'],
+    },
+    {
+        title: 'Dành cho nhà tuyển dụng',
+        description: ['Đăng tuyển dụng', 'Quản lý hồ sơ', 'Sản phẩm dịch vụ khác', 'Liên hệ'],
+    },
+    {
+        title: 'Giấy phép',
+        description: ['Điều khoản bảo mật', 'Điều khoản sử dụng'],
+    },
+];
+
+export default function Footer() {
+    const classes = useStyles();
+
+    return (
+        <React.Fragment>
```

Comment:
by modern convention, use <> </> for react fragment or div tag if it have className